projects/imagenet/experiments_superconvergence.py

```diff
@@ -126,92 +128,60 @@
         anneal_strategy="linear",
     ),
     num_classes=100,
-
-    # Create default sparse network
     model_args=dict(config=dict(num_classes=100, defaults_sparse=False)),
 )
 
-FASTAI10 = copy.deepcopy(DEFAULT)
-FASTAI10.update(
-    epochs=50,
-    num_classes=10,
+FASTAI1000 = copy.deepcopy(FASTAI100)
+FASTAI1000.update(
+    epochs=35,
+    num_classes=1000,
```

Comment:
We dont need to set num_classes to 1000. It is the default value

Reply:
It's there because the parameters are inherited from the 100 category config.

Reply:
makes sense